src/packages/recompose/componentFromStream.js

```diff
@@ -3,8 +3,18 @@ import { createChangeEmitter } from 'change-emitter'
 import $$observable from 'symbol-observable'
 import { config as globalConfig } from './setObservableConfig'
 
-export const componentFromStreamWithConfig = config => propsToVdom =>
-  class ComponentFromStream extends Component {
+const identity = v => v
+
+export const createObservableConfig = rawConfig => ({
```

Comment:
Why? What the reason to change current implementation? No need to merge the config as in recompose implementation `toESObservable`,`fromESObservable`  are defined.
It's not the `options` object with tens of props.